test/infrastructure/docker/api/v1beta1/condition_consts.go

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 package v1beta1
 
 import (
-	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta2"
+	clusterv1 "sigs.k8s.io/cluster-api/api/core/v1beta2"
```

Comment:
I guess moving these was out of scope? (would have been `/test/api/core/v1beta1` of course)